### ISSUE TITLE
Preferences UI: Wrong default export format selected

### ISSUE THREAD

**Issue by swaldman3** [codinguser/gnucash-android#839]
#### Steps to reproduce the behaviour
* Go to menu -> Settings -> Backup & Export -> Default Export Format.
* Select an option
- Note the option that was selected vs the selected option that then appears in the Settings menu.

#### Expected behaviour
* The option that appears in the settings menu should be the one selected, and should also be the format used for export

#### Actual behaviour
* If "QIF" is selected, "CSVT" appears.
* If "OFX" is selected, "QIF" appears.
* If "XML" is selected, "OFX" appears.
In other words the value actually chosen is one item higher up the list than the option that is tapped on. The format used for export appears to be the one shown as selected, not the one that is tapped on.

#### Software specifications
* GnuCash Android version: 2.4.0
* System Android version: 9
* Device type: Nokia 7.1 (model TA-1085)
